core/src/util/builders.cpp

```diff
@@ -117,12 +116,10 @@ void Builders::buildPolygonExtrusion(const Polygon& _polygon, float _minHeight,
         size_t lineSize = line.size();
         sumIndices += lineSize * 6;
         sumVertices += (lineSize - 1) * 4;
-
-        _ctx.numVertices = sumVertices;
     }
```

Comment:
Sure, I wanted to get the fix in asap before I left for the day!

Will update this PR to remove any size pre-computation.